common/git-hooks/pre-commit

```diff
@@ -6,4 +6,8 @@
 # Invoke the "rush prettier" custom command to reformat files whenever they
 # are committed. The command is defined in common/config/rush/command-line.json
 # and uses the "rush-prettier" autoinstaller.
-node common/scripts/install-run-rush.js prettier || exit $?
\ No newline at end of file
+node common/scripts/install-run-rush.js prettier || exit $?
+if [ "$SKIP_GITLEAKS" != 1 ] 
+then
+   gitleaks protect -v --staged --report-path ./findings.json
```

Comment:
This just fails without gitleaks installed. Can we detect that and issue a warning or suggest to skip the check if not found?

Reply:
Made it a bit fancier as well